src/mono/wasm/runtime/driver.c

```diff
@@ -470,7 +470,7 @@ mono_wasm_load_runtime (const char *unused, int debug_level)
 	// We should enable this as part of the wasm build later
 #ifndef DISABLE_THREADS
 	monoeg_g_setenv ("MONO_THREADS_SUSPEND", "coop", 0);
-	monoeg_g_setenv ("MONO_SLEEP_ABORT_LIMIT", "250", 0);
+	monoeg_g_setenv ("MONO_SLEEP_ABORT_LIMIT", "50000", 0);
```

Comment:
is this intended? It feels like too large of a value for normal use, but 250 is probably too low